redaxo/src/core/lib/util/finder.php

```diff
@@ -185,7 +185,7 @@ public function sort($sort = rex_sortable_iterator::KEYS)
     }
 
     /**
-     * @return Iterator|SplFileInfo[]
+     * @return RecursiveCallbackFilterIterator|RecursiveIteratorIterator|rex_sortable_iterator
```

Comment:
Hmmm?

Reply:
`SplFileInfo[]` ist wichtig, damit man in foreach dann die korrekte autovervollständigung bekommt.
(auch wenn phpstorm da zurzeit leider einen bug hat für die kombi ::factory() und dann getIterator mit Elementtypangabe)